app/src/main/AndroidManifest.xml

```diff
@@ -54,7 +54,7 @@
             android:label="@string/title_activity_signup"
             android:theme="@style/Theme.CoachMe" />
         <activity
-            android:name=".ScheduleActivity"
+            android:name=".schedule.ScheduleActivity"
```

Comment:
great idea of moving this activity into a separate package, this makes it much easier to get an overview of the project parts!